content/en/docs/Concepts/olm-architecture/operator-catalog/creating-an-update-graph.md

```diff
@@ -255,6 +255,308 @@ entries:
+
+As QE / automation has time, updates are tested from older releases. Those extra edges will be added over time.
+
+We'll assume a semver, kube-like versioning/update strategy.
+
+#### By only adding Skips
+
+---
+`Installed` is at `v1`:
+
+{{<mermaid>}}
+graph LR;
+  I(Installed - v1) 
+{{</mermaid>}}
+---
+Release `v1.1` that `replaces` `v1`:
+
+{{<mermaid>}}
+graph LR;
+  I(Installed - v1)
+  subgraph Update<br>Options
+    v1.1
+  end
+  v1.1 -->|replaces| I
+{{</mermaid>}}
+---
+Release `v1.2` that `replaces` `v1.1`:
+
+{{<mermaid>}}
+graph LR;
+  I(Installed - v1)
+  subgraph Update<br>Options
+    v1.1
+  end
+  v1.1 -->|replaces| I
+  v1.2 -->|replaces| v1.1
+{{</mermaid>}}
+---
+Test that `v1.2` can also update from `v1`. Add a `skips: v1` to `v1.2`:
+
+{{<mermaid>}}
+graph LR;
+  I(Installed - v1)
+  subgraph Update<br>Options
+    v1.2
+    v1.1
+  end
+  v1.1 -->|replaces| I
+  v1.2 -->|replaces| v1.1
+  v1.2 -->|skips| I
+{{</mermaid>}}
+
+---
+Release `v2` that `replaces` `v1.2`:
+
+{{<mermaid>}}
+graph LR;
+  I(Installed - v1)
+  subgraph Update<br>Options
+    v1.1
+    v1.2
+  end
+  v1.1 -->|replaces| I
+  v1.2 -->|replaces| v1.1
+  v1.2 -->|skips| I
+  v2 --> |replaces| v1.2
+  
+{{</mermaid>}}
+---
+
+Test that `v2` can also update from `v1.1` and `v1`. Add `skips: [v1, v1.1]`:
+
+{{<mermaid>}}
+graph LR;
+  I(Installed - v1)
+  subgraph Update<br>Options
+    v2
+    v1.2
+  end
+  v1.1 -->|replaces| I
+  v1.2 -->|replaces| v1.1
+  v1.2 -->|skips| I
+  v2 --> |replaces| v1.2
+  v2 --> |skips| v1.1
+  v2 --> |skips| I
+{{</mermaid>}}
+
+Note that `v1.1` is no longer an option for `v1` - it has been skipped by something else in the channel.
+
+---
+
+Release `v2.1` that replaces `v2`
+
+{{<mermaid>}}
+graph LR;
+  I(Installed - v1)
+  subgraph Update<br>Options
+    v2
+    v1.2
+  end
+  v1.1 -->|replaces| I
+  v1.2 -->|replaces| v1.1
+  v1.2 -->|skips| I
+  v2 --> |replaces| v1.2
+  v2 --> |skips| v1.1
+  v2 --> |skips| I
+  v2.1 --> |replaces| v2
+{{</mermaid>}}
+
+---
+
+Test that `v2.1` can also update from `v1.2`. Add `skips: [v1.2]`:
+
+{{<mermaid>}}
+graph LR;
+  I(Installed - v1)
+  subgraph Update<br>Options
+    v2
+  end
+  v1.1 -->|replaces| I
+  v1.2 -->|replaces| v1.1
+  v1.2 -->|skips| I
+  v2 --> |replaces| v1.2
+  v2 --> |skips| v1.1
+  v2 --> |skips| I
+  v2.1 --> |replaces| v2
+  v2.1 --> |skips| v1.2
+{{</mermaid>}}
+
+---
+
+Add a `v2.2` that `replaces` `v2.1` and `skips: [v2]`
+
+{{<mermaid>}}
+graph LR;
+  I(Installed - v1)
+  v1.1 -->|replaces| I
+  v1.2 -->|replaces| v1.1
+  v1.2 -->|skips| I
+  v2 --> |replaces| v1.2
+  v2 --> |skips| v1.1
+  v2 --> |skips| I
+  v2.1 --> |replaces| v2
+  v2.1 --> |skips| v1.2
+  v2.2 --> |replaces| v2.1
+  v2.2 --> |skips| v2
+{{</mermaid>}}
+
+Note that `v1` is now completely cut off from updates: there is no version with an edge to `v1` that is not `skipped` by something else.
```

Comment:
Same comment here, I'm not sure what this means? 

> there is no version with an edge to `v1`

But I can see an edge from v1.1 to v1 for example. Can I not upgrade to v1.1 from v1 since v1.1 replaces v1? 

What is this "something else"?